### DISCUSSION TITLE
Allow embedding HTML code in Custom Links

### DISCUSSION THREAD

**Discussion by helfrichmichael** [JulianPrieber/littlelink-custom#685] [Feature requests]
It seems like in a recent version (downloaded 4.7.1 today) there is no longer the ability to copy/paste HTML code into the Custom Links element. Now if I paste it, it just pastes as plain text. Looking at some of the other editors (IE for footer pages) it allows HTML code. Can we please swap the editor to allow HTML code to be embedded? This is useful for Shopify and embedding needs.

**Comment by helfrichmichael** [comment]
Note: If you manually edit the link the SQLite or MySQL database, you can just paste the raw HTML onto the row and it works as expected.

**Comment by JulianPrieber** [comment]
We won't allow any sort of custom HTML due to security reasons.

**Reply by MyBlood** [reply]
> We won't allow any sort of custom HTML due to security reasons.

Hi. But how do I insert a video player from an unsupported service?
